utils/node_local_test.py

```diff
@@ -2662,10 +2662,7 @@ def test_cont_copy(self):
         src_dir = tempfile.TemporaryDirectory(prefix='copy_src_',)
         with open(join(src_dir.name, 'file'), 'w') as ofd:
             ofd.write('hello')
-        cwd = os.getcwd()
-        os.chdir(src_dir.name)
-        os.symlink('file', 'file_s')
-        os.chdir(cwd)
+        os.symlink(join(src_dir.name, 'file'), join(src_dir.name, 'file_s'))
```

Comment:
You only need the join on the source.  Without it you'd have a relative link, with it you'll have an absolute link, and daos might treat them differently.  @daltonbohning would know.

Reply:
I printed out the current working directory. It is not under the directory daos mounted at. If I use relative path 'file_s' for symlink(), the symbolic link is created under an undesired directory.

Reply:
Just updated the code. os.symlink('file', join(src_dir.name, 'file_s')) should work. It creates a relative path.

Reply:
Links are tricky...
In DFUSE, absolute and relative links mostly work the way you expect them to - absolute links are absolute from the **_root of the encompassing filesystem_**. - because each entry in a path is handled/looked-up individually.
In DFS, absolute links are with respect to the **_container root_** (DFS only knows about DFS), so dereferencing `join(src_dir.name, 'file')` would attempt a lookup with respect to the _**container root**_, which is most likely invalid.

For this link, using a relative path will work with both DFUSE and DFS. However, broken symlinks are fine as long as they aren't dereferenced.